### PR TITLE
Ditch pykafka

### PR DESCRIPTION
**Pull request by swenzel**

### CODE REVIEW

**Review by ognjen-j — DISMISSED** [review]

**Review by Crandel — DISMISSED** [review]

**Comment by hfjn** [comment]
Very cool. How are the performance improvements?

**Review by hfjn — APPROVED** [review]

File: esque/controller/topic_controller.py [hunk 1]
```diff
@@ -92,8 +66,14 @@ def create_topics(self, topics: List[Topic]):
             )
             future_list = self.cluster.confluent_client.create_topics([new_topic], operation_timeout=60)
             ensure_kafka_future_done(next(islice(future_list.values(), 1)))
+            for _ in range(80):
+                topic_data = self.cluster.confluent_client.list_topics(topic=topic.name).topics[topic.name]
+                if topic_data.error is None:
+                    break
+                time.sleep(0.125)
+            else:
```
Comment (hfjn) [inline esque/controller/topic_controller.py:74]: Wohoo. Finally a use for a for-else loop.

Comment (swenzel) [reply esque/controller/topic_controller.py:74]: They're really convenient for timeouts like this or for lookups :slightly_smiling_face:

Comment (hfjn) [reply esque/controller/topic_controller.py:74]: Yeah. The lookup use case is super handy.

**Comment by swenzel** [comment]
> Very cool. How are the performance improvements?

The tests suite went down from ~18 minutes to ~6 minutes. Not sure if it's completely equivalent to everyday use, though. We'll see :slightly_smiling_face:

**Pull request merged**